components/currentAgenda.tsx

```diff
@@ -25,10 +35,23 @@ export const CurrentAgenda: React.FC<CurrentAgendaProps> = ({
   acceptingFeedback,
   feedbackLink,
 }) => {
+  const { IsInProgress } = getConferenceDates(Conference, dateTimeProvider.now())
+
   return (
     <Fragment>
       <p>Tap on a session to see more details&hellip;</p>
 
+      {IsInProgress && (
```

Comment:
@antonjb This should actually be `IsAcceptingFeedback`

Reply:
Yep, you're right. Have addressed in #222